src/ui/components/NewProject/index.jsx

```diff
@@ -1,5 +1,189 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+
+import Button from "../shared/Button";
+import ToastPopup from "../shared/Toast";
+
+import useProjectStore from "../../../store/project";
+import useProjectVersionStore from "../../../store/projectVersion";
+import getProjectKeyFromURI from "../../../utils/getProjectKeyfromURI";
+import getAllVersions from "../../../service/getAllVersions";
+import isValidFigmaUrl from "../../../utils/isValidFigmaUrl";
+import postMessage from "../../../utils/postMessage";
```

Comment:
- `P1`
```suggestion
import useProjectStore from "../../../store/project";
import useProjectVersionStore from "../../../store/projectVersion";

import getProjectKeyFromURI from "../../../utils/getProjectKeyfromURI";
import isValidFigmaUrl from "../../../utils/isValidFigmaUrl";
import postMessage from "../../../utils/postMessage";
import getAllVersions from "../../../service/getAllVersions";
```

`store import`구문에서 한줄 개행 해주시고, `utils` 끼리 묶어서 작성하면 더욱 가독성이 좋을 것 같습니다!

Reply:
네 반영하겠습니다! 👍